api_app/analyzers_manager/observable_analyzers/malprob.py

```diff
@@ -0,0 +1,137 @@
+import requests
+
+from api_app.analyzers_manager import classes
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+
+class MalprobSearch(classes.ObservableAnalyzer):
+    url: str = "https://malprob.io/api"
+
+    def update(self):
+        pass
+
+    def run(self):
+        response = requests.get(
+            f"{self.url}/search/{self.observable_name}",
+            timeout=10,
+        )
+        response.raise_for_status()
+        if response.status_code == 204:
+            self.disable_for_rate_limit()
+            raise classes.AnalyzerRunException("Limit reached for API")
+        return response.json()
```

Comment:
so this does not require an api key but there's still the disable for rate limit thing? I don't get it

Reply:
Yes, mb, this shouldn't be here. Pushing a fix.